Examples/VaporExample/README.md

```diff
@@ -8,22 +8,22 @@ This application is intended to demonstrate best practices for integrating the d
 
 The application contains both a REST API as well as a minimal frontend built with Vapor's templating language [Leaf](https://github.com/vapor/leaf).
 
-This application require Swift 5.2 and MongoDB 3.6+. It will run on Linux as well as macOS 10.15+.
+This application requires Swift 5.5+ and MongoDB 3.6+. It will run on Linux as well as macOS 12+.
+
+If you are looking for an example application that uses older Swift versions prior to the introduction of structured concurrency, please refer to a previous version of this application and README [here](https://github.com/mongodb/mongo-swift-driver/tree/5d9fad121d7cb3ded61087de300ff007766ccd55/Examples/VaporExample).
 
 ## Building and Running the Application
 1. Install MongoDB on your system if you haven't already. Downloads are available [here](https://www.mongodb.com/download-center/community).
 1. Start up MongoDB running locally: `mongod --dbpath some-directory-here`. You may need to specify a `dbpath` directory for the database to use.
 1. Run `./loadData.sh` to load example application data into the database.
-1. Install Swift 5.2+ on your system if you haven't already. You can download Swift and find instructions for installing it [here](https://swift.org/download/).
+1. Install Swift 5.5+ on your system if you haven't already. You can download Swift and find instructions for installing it [here](https://swift.org/download/).
 1. From the root directory of the project, run `swift build`. This will likely take a while the first time you do so.
 1. Once building has completed, run `swift run` from the root directory. You should get a message that the server has started running on `http://127.0.0.1:8080`.
 1. Open up your browser and visit `http://127.0.0.1:8080`. You should see the application and be able to test out adding, deleting, and editing data in the collection.
 
 ## Application Architecture
 
-This is a fully asynchronous application. At its core is [SwiftNIO](https://github.com/apple/swift-nio), which is used to implement both Vapor and the MongoDB driver.
```

Comment:
users don't really need to think about NIO as much now (and in fact can completely ignore it when using MongoDBVapor since we handle setting up the client and passing in the EventLoopGroup for them), so it didn't seem necessary to still call this out